pkg/apis/minio.min.io/v1/constants.go

```diff
@@ -112,23 +112,23 @@ const ConsoleTenantLabel = "v1.min.io/console"
 // MCSPort specifies the default Console port number.
```

Comment:
should this also be modified?

Reply:
let's rename the `mcs` project to console as well :-)

Reply:
Not resolving mcs project name in this PR :-)
Fixed the port

Reply:
That is done mcs -> console